app/packages/looker/src/elements/common/error.ts

```diff
@@ -25,6 +34,12 @@ export class ErrorElement<State extends BaseState> extends BaseElement<State> {
       this.errorElement.classList.add(lookerErrorPage);
       const errorImg = document.createElement("img");
       errorImg.src = errorIcon;
+      errorImg.title = "Click to reload";
```

Comment:
I haven't been able to trigger the error myself, but looks like here you've added a tooltip and changed the cursor style on hover for the `!` icon.

That's good, but I wonder if users still may not deduce that they can click to refresh. What if the image also changed from a `!` to a refresh icon on hover?

Reply:
Done